config-linux.md

```diff
@@ -10,18 +10,18 @@ Applications expecting a Linux environment will very likely expect these file pa
 
 The following filesystems SHOULD be made available in each container's filesystem:
 
-|   Path   |  Type  |
+| Path     | Type   |
 | -------- | ------ |
-| /proc    | [proc](https://www.kernel.org/doc/Documentation/filesystems/proc.txt)   |
-| /sys     | [sysfs](https://www.kernel.org/doc/Documentation/filesystems/sysfs.txt)   |
-| /dev/pts | [devpts](https://www.kernel.org/doc/Documentation/filesystems/devpts.txt) |
-| /dev/shm | [tmpfs](https://www.kernel.org/doc/Documentation/filesystems/tmpfs.txt)   |
+| /proc    | [procfs][procfs]   |
+| /sys     | [sysfs][sysfs]     |
+| /dev/pts | [devpts][devpts]   |
+| /dev/shm | [tmpfs][tmpfs]     |
 
 ## Namespaces
 
 A namespace wraps a global system resource in an abstraction that makes it appear to the processes within the namespace that they have their own isolated instance of the global resource.
 Changes to the global resource are visible to other processes that are members of the namespace, but are invisible to other processes.
-For more information, see [the man page](http://man7.org/linux/man-pages/man7/namespaces.7.html).
+For more information, see the [NAMESPACES.7][namespaces.7_2] man page.
```

Comment:
Man pages seem to be mostly referenced in lowercase.  So despite the `NAMESPACES(7)` at the top of `namespaces(7)`, I'd rather call it `namespaces(7)` to match the “SEE ALSO” references at the end of `user_namespaces(7)`, etc.  No need to shout here ;).